view-binding-utils/src/main/java/com/airbnb/mvrx/viewbinding/FragmentViewBindingDelegate.kt

```diff
@@ -21,16 +23,18 @@ class FragmentViewBindingDelegate<T : ViewBinding>(
     private val bindMethod = bindingClass.getMethod("bind", View::class.java)
 
     init {
-        fragment.viewLifecycleOwnerLiveData.observe(fragment) { viewLifecycleOwner ->
-            viewLifecycleOwner.lifecycle.addObserver(object : LifecycleObserver {
-                @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
-                fun onDestroy() {
-                    // Lifecycle listeners are called before onDestroyView in a Fragment.
-                    // However, we want views to be able to use bindings in onDestroyView
-                    // to do cleanup so we clear the reference one frame later.
-                    clearBindingHandler.post { binding = null }
-                }
-            })
+        fragment.lifecycleScope.launch {
```

Comment:
why did you need this change?

Reply:
this is the fix mentioned in the description "Also includes a fix for FragmentViewBindingDelegate which can try to observe on a background thread causing a crash. (reproed in sample app launcher)"